### PR TITLE
feat: add shareNFTOnTwitter to dropdown

### PR DESCRIPTION
**Pull request by alantoa**
# Why

we want open the Share feature again.

# How

open Share again and add `shareNFTOnTwitter` hooks easy to share web pages to Twitter.

image

# Test Plan

check if the `Share` feature works.

### CODE REVIEW

File: packages/app/hooks/use-share-nft.ts [hunk 1]
```diff
@@ -1,29 +1,49 @@
+import { Linking } from "react-native";
+
 import { useShare } from "app/hooks/use-share";
 import { track } from "app/lib/analytics";
 import { CHAIN_IDENTIFIERS } from "app/lib/constants";
 import { NFT } from "app/types";
 
+import { getTwitterIntent } from "../utilities";
```
Comment (intergalacticspacehighway) [inline packages/app/hooks/use-share-nft.ts]: nit: app/utilities (ocd)

File: packages/app/hooks/use-share-nft.ts [hunk 2]
```diff
@@ -1,29 +1,48 @@
...
+      nft?.chain_identifier
+    )}/${nft?.contract_address}/${nft?.token_id}`;
+    // Todo: add share Claim/Drop copytext
+    Linking.openURL(
+      getTwitterIntent({
+        url,
+        message: ``,
+      })
+    );
+    track("NFT Shared on Twitter", { type: "Twitter" });
```
Comment (axeldelafosse) [inline packages/app/hooks/use-share-nft.ts]: `NFT Shared on Twitter` should stay `NFT Shared`. Adding the `type` is perfect, don't need to create another event.

**Review by axeldelafosse — APPROVED** [review]
LFG ✨

File: packages/app/hooks/use-share-nft.ts (hunk 1 above)
Comment (axeldelafosse) [reply packages/app/hooks/use-share-nft.ts]: Haha it's not a nitpick, no worries. Can we find a way to get absolute imports with auto import on VS Code? I suspect this was added by auto import, right?

**Review by intergalacticspacehighway — APPROVED** [review]
LGTM ❇️

File: packages/app/hooks/use-share-nft.ts (hunk 1 above)
Comment (alantoa) [reply packages/app/hooks/use-share-nft.ts]: yeah it's auto import, using absolute paths will solve it!

**Pull request merged**